runtime/common/src/constants.rs

```diff
@@ -397,4 +397,9 @@ parameter_types! {
 	pub const CapacityPerToken: Perbill = Perbill::from_percent(2);
 	pub const CapacityRewardCap: Permill = Permill::from_parts(3_800);  // 0.38% or 0.0038 per RewardEra
 }
+pub type CapacityRewardEraLength =
```

Comment:
Marginally related - pulled this out so it could be different for different environments.